lib/mix/lib/mix/task.ex

```diff
@@ -7,11 +7,11 @@ defmodule Mix.Task do
   in a module starting with `Mix.Tasks.` and defining
   the `run/1` function:
 
-      defmodule Mix.Tasks.Hello do
+      defmodule Mix.Tasks.Echo do
         use Mix.Task
 
-        def run(_args) do
-          Mix.shell.info "hello"
+        def run(args) do
+          Mix.shell.info Enum.join(args, " ")
```

Comment:
It is an **example**. It is not meant to be a functional implementation. Wrapping every argument in a quote is going to be more confusing because the user most likely won't try to pass quoted arguments. Plus the behaviour here is exactly what `echo` in your OS does:

```
$ echo "foo bar" 1
foo bar 1
```